pkg/aws/ec2.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const FILTER_NAME_RESOURCEID string = "resource-id"
+const FILTER_NAME_RESOURCETYPE string = "resource-type"
+const FILTER_VALUE_INSTANCE string = "instance"
```

Comment:
Nit: For whatever it's worth, this will trigger a `don't use ALL_CAPS in Go names; use CamelCase` in common Go linters.

It could also be re-written as...

```golang
const (
  FILTER_NAME_RESOURCEID      string = "resource-id"
  FILTER_NAME_RESOURCETYPE string = "resource-type"
  FILTER_VALUE_INSTANCE           string = "instance"
)
```

Or perhaps be a `FilterName` enum a la...

```golang
type FilterName string

const (
  ResourceID FilterName = "resource-id"
  ...
)
```

Reply:
I switched it to be camelCased since these should not be used outside the package.